docs/sources/release-notes/upgrading-helm-from-2.1-to-3.0.md

```diff
@@ -0,0 +1,317 @@
+          insecure: true
+    {{- end }}
+    auth:
+      type: enterprise
+    cluster_name: "{{ .Release.Name }}"
+    license:
+      path: "/license/license.jwt"
+    {{- if .Values.gateway.useDefaultProxyURLs }}
+    gateway:
+      proxy:
+        default:
+          url: http://{{ template "mimir.fullname" . }}-admin-api.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        admin_api:
+          url: http://{{ template "mimir.fullname" . }}-admin-api.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        alertmanager:
+          url: http://{{ template "mimir.fullname" . }}-alertmanager.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        compactor:
+          url: http://{{ template "mimir.fullname" . }}-compactor.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        distributor:
+          url: dns:///{{ template "mimir.fullname" . }}-distributor-headless.{{ .Release.Namespace }}.svc.{{ .Values.global.clusterDomain }}:{{ include "mimir.serverGrpcListenPort" . }}
+        ingester:
+          url: http://{{ template "mimir.fullname" . }}-ingester.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        query_frontend:
+          url: http://{{ template "mimir.fullname" . }}-query-frontend.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        ruler:
+          url: http://{{ template "mimir.fullname" . }}-ruler.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+        store_gateway:
+          url: http://{{ template "mimir.fullname" . }}-store-gateway.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}
+    {{- end }}
+    instrumentation:
+      enabled: true
+      distributor_client:
+        address: 'dns:///{{ template "mimir.fullname" . }}-distributor-headless.{{ .Release.Namespace }}.svc.{{ .Values.global.clusterDomain }}:{{ include "mimir.serverGrpcListenPort" . }}'
+    {{- end }}
+```
+
+After applying the migration steps listed in this guide, the equivalent `values.yaml` for version 3.0.0 looks like this:
+
+First we create an external secret to store the credentials:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+    name: mimir-bucket-secret
+data:
+    AWS_ACCESS_KEY_ID: FAKEACCESSKEY
+    AWS_SECRET_ACCESS_KEY: FAKESECRETKEY
+```
+
+```yaml
+rbac:
+  create: false
+
+chunks-cache:
+  enabled: true
+  replicas: 1
+
+index-cache:
+  enabled: true
+  replicas: 1
+
+metadata-cache:
+  enabled: true
+  replicas: 1
+
+results-cache:
+  enabled: true
+  replicas: 1
+
+metaMonitoring:
+  serviceMonitor:
+    enabled: true
+
+mimir:
+  structuredConfig:
+    blocks_storage:
+      backend: s3
+      s3:
+        endpoint: s3.amazonaws.com
+        bucket_name: my-blocks-bucket
+        access_key_id: ${AWS_ACCESS_KEY_ID}
+        secret_access_key: ${AWS_SECRET_ACCESS_KEY}
```

Comment:
nitpick, due to the transformation before the configuration is written to the ConfigMap, helm will put everything in alphabetic order, so to avoid confusion:

```suggestion
        access_key_id: ${AWS_ACCESS_KEY_ID}
        bucket_name: my-blocks-bucket
        endpoint: s3.amazonaws.com
        secret_access_key: ${AWS_SECRET_ACCESS_KEY}
```